tests/cucumber.rs

```diff
@@ -23,27 +23,27 @@ pub struct MyWorld {
 #[given("the plane is flying")]
 fn is_flying(_world: &mut MyWorld) {}
 
-#[given(regex = r#"^(.+) is (.*)armed$"#)]
+#[given(regex = r#"^(.+) is ?(not)? armed$"#)]
 fn is_armed(world: &mut MyWorld, alert: AlertWrapper, maybe_not: String) {
-    if maybe_not.starts_with("not") {
+    if maybe_not.eq("not") {
```

Comment:
Would it be possible to use `maybe_not == "not"` here? If yes, what is the pro/con of doing it?

Reply:
It would be possible and I do not see any pro/cons. This is personal preference

Reply:
Then I would like to have the `==`. I think it looks cleaner to use an operator than a method call. Can you update the PR accordingly?

Reply:
@Steavo005 ping

Reply:
Changed with e17de73

Reply:
@steav005 sorry, but can squash both commits into one? Thank you!

Reply:
Done 9fd41ea